timemachine/cpp/src/barostat.cu

```diff
@@ -172,41 +202,115 @@ void __global__ find_group_centroids(
     atomicAdd(centroids + mol_idx*3+2, FLOAT_TO_FIXED<double>(coords[atom_idx*3+2]));
 }
 
-void MonteCarloBarostat::reset_counters() {
-    num_attempted_ = 0;
-    num_accepted_ = 0;
+void __global__ k_setup_barostat_move(
+    const double * rand, // [2], use first value, second value is metropolis condition
```

Comment:
any reason why this isn't __restrict__'d?

Reply:
Nope, will add